nuxt.config.ts

```diff
@@ -94,6 +94,7 @@ export default defineNuxtConfig({
       githubOrg: '',
       githubEnt: '',
       githubTeam: '',
+      enterpriseType: 'copilot-only',  // can be overridden by NUXT_PUBLIC_ENTERPRISE_TYPE environment variable
```

Comment:
I think default should be 'full'

Reply:
I agree too, but i left like it in order to have the same behavior as the current implementation. Anyway, this new flag could be documented for deployment.